.evergreen/init.sh

```diff
@@ -190,3 +190,70 @@ _init_sh_evg_dir="$(dirname "${_init_sh_this_file}")"
 # this one, and will therefore remain as native paths
 EVG_DIR="$(native_path "${_init_sh_evg_dir}")"
 LIBMONGOCRYPT_DIR="$(dirname "${EVG_DIR}")"
+
+is_true() {
+    declare var="$1"
+    declare val="${!var-}"  # Default is '' empty
```

Comment:
Why the need for `!` in `${!var-}`?

Reply:
An indirect dereference. You pass the name of the var, this will dereference the var by name.

Reply:
Oh, I overlooked the intended usage of this function as verifying the value of the given _variable_ has a truthy value, not the argument itself. Disregard.